apps/ui/src/composables/useSpaceAlerts.ts

```diff
@@ -89,6 +94,48 @@ export function useSpaceAlerts(
       .filter(network => !!network);
   });
 
+  const isProExpiringSoon = computed(() => {
+    // Check if expiration date exists and is valid (greater than 0)
+    if (!space.value.turbo_expiration || space.value.turbo_expiration === 0) {
+      return false;
+    }
+
+    const now = Date.now();
+    const expirationTime = space.value.turbo_expiration * 1000; // Convert to milliseconds
+    const warningThresholdMs =
+      PRO_EXPIRATION_WARNING_DAYS * 24 * 60 * 60 * 1000;
+
+    // Check if expiration is in the future and within warning threshold
+    return expirationTime > now && expirationTime - now <= warningThresholdMs;
+  });
+
+  const daysUntilExpiration = computed(() => {
+    if (!space.value.turbo_expiration || space.value.turbo_expiration === 0) {
+      return 0;
+    }
+
+    const now = Date.now();
+    const expirationTime = space.value.turbo_expiration * 1000;
+    const diff = expirationTime - now;
+
+    return Math.ceil(diff / (24 * 60 * 60 * 1000));
+  });
+
+  const isProJustExpired = computed(() => {
+    // Check if expiration date exists and is valid (greater than 0)
+    if (!space.value.turbo_expiration || space.value.turbo_expiration === 0) {
```

Comment:
```suggestion
    if (!space.value.turbo_expiration) {
```

I think the second one is redundant, same on line 99